src/post/AuthorBio.js

```
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import steemdb from 'steemdb';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Link } from 'react-router';
+import Avatar from '../widgets/Avatar';
+import Loading from '../widgets/Loading';
+import Follow from '../widgets/Follow';
+import { followUser, unfollowUser, getFollowing } from '../user/userActions';
+
+@connect(
+  ({ auth, userProfile }) => ({ auth, following: userProfile.following }),
+  dispatch => bindActionCreators({ followUser, unfollowUser, getFollowing }, dispatch)
+)
+
+class AuthorBio extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      author: null,
+    };
+  }
+
+  componentDidMount() {
+    const loggedInUser = this.props.auth.user.name;
+    this.props.getFollowing(loggedInUser);
+    steemdb.accounts({
+      account: this.props.authorName
+    }, (err, result) => {
+      if (result.length) {
+        const author = result[0];
+
+        try {
+          author.json_metadata = JSON.parse(result[0].json_metadata);
+        } catch (e) {
+          author.json_metadata = {};
```

better to throw an error on catch:

```
throw new Error(`Error parsing jsonMetadata for user ${username}`);
```